spec/project_version_tag_spec.rb

```diff
@@ -2,7 +2,7 @@
 
 describe Jekyll::ProjectVersionTag do
   let(:git_test_command) { 'git rev-parse' }
-  let(:git_tag_command) { ' git describe --tags ' }
+  let(:git_tag_command) { ' git describe --tags --always ' }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.